SampleScripts/PowerManagement-ManagementDomain.ps1

```diff
@@ -681,7 +680,7 @@ if ($PsBoundParameters.ContainsKey("shutdown") -or $PsBoundParameters.ContainsKe
                         Write-PowerManagementLogMessage -Type INFO -Message "End of the shutdown sequence!" -Colour Green
                         Exit
                     } else {
-                        Start-Sleep $sleepTime
+                        Start-Sleep -s $sleepTime
```

Comment:
`$sleepTime` or `$sleep_time`? I see both used.

Reply:
addressed in commit a0e6a231b99ac37596cd1b07cf674e22d71bda11 and e0329cefcd79a66e5c7c1a206cebb4e0f0d3e3f1